www/data_service/src/services/stream/stream.service.coffee

```diff
@@ -0,0 +1,36 @@
+class Stream extends Factory
+    constructor: ($log) ->
+        return class StreamInstance
```

Comment:
I've got the idea from here: https://github.com/CaryLandholt/ng-classify#factory